src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return sum - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return sum - 20;
  } else if (days >= 7) {
```
use `if - else if` construction